src/components/v2v3/V2V3Project/V2V3Project.tsx

```diff
@@ -170,7 +176,21 @@ export function V2V3Project({
 
   // Change URL without refreshing page
   const removeQueryParams = () => {
-    router.push(v2v3ProjectRoute({ projectId }), undefined, { shallow: true })
+    const newQuery: Record<string, string> = {}
```

Comment:
What does this do?

Reply:
This was a leak from another PR. Out of the diff now